### ISSUE TITLE
Optional is a restricted API in this lab? (my lab8 runs on optionals...)

### ISSUE THREAD

**Issue by ChoonLeng773** [nus-cs2030/2122-s2#276]
# Summary

Description of the question / problem here.

## Description:

Clarifications or details with respect to the issue you are facing

## Code Snippets

Attach code snippets (if any)

## Screenshots

Screenshots (if any)
<img width="273" alt="image" src="https://user-images.githubusercontent.com/73415759/162153489-1282ad24-01ce-49df-9b99-4411b6218fc1.png">

**Comment by pohja1999** [comment]
I think you wanna check if you used get/ispresent or empty methods of opt😨

**Comment by lukasleeyo** [comment]
Yup! The methods ``get()``, ``isPresent()`` and ``isEmpty()`` under ``Optional`` class is banned by the codecrunch. You may consider using ``orElse(obj)`` or ``orElseThrow()`` if you are trying to use ``get()``.

**Comment by alwinngjw** [comment]
Hi, yea i think the 3 methods get(), isPresent() and isEmpty() is not allowed by code cruch.

Maybe using orElse() may solve your issue

Hope this helps

**Comment by ReganKvioke** [comment]
I think you used get(), isPresent() or isEmpty(). Probably a small mistake somemore. Hope this helps. Good luck

**Comment by jamietan123** [comment]
You can use orElse(), orElseGet() or orElseThrow().

**Comment by yizziepeasy** [comment]
Hi, the methods get(), isPresent() and isEmpty() under the Optional class is banned by codecrunch as emphasised by Prof Henry. You may consider using Optional(1).orElse(x) which returns the value of Optional since it is not empty, if not it returns x.

**Comment by sashimitrash** [comment]
Yup!  isPresent() and isEmpty() all make the use of optional class obstinate, so it is banned from codecrunch.

**Comment by lvxhnat** [comment]
yeap, had the same issue as well, as stated above, using .ofNullable, .empty, but issue turned out to be .get() on an optional instance, which was banned
